backend/api/models/inbox.py

```diff
@@ -0,0 +1,33 @@
+from django.db import models
+import uuid
+
+# Related Models
+from api.models.author import Author
+from api.models.post import Post
+from api.models.comment import Comment
+from api.models.like import Like
+
+MessageTypes = [
+    ('post', 'post'),
+    ('comment', 'comment'),
+    ('like', 'like'),
+]
+
+class Inbox(models.Model):
+    # Unique ID of this inbox record
+    item_id     = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False, unique=True)
+    
+    # What type of message is in the inbox
+    # THIS SHOULD NEVER BE NULL, BUT WE ALSO DONT HAVE A DEFAULT VALUE
```

Comment:
The inbox should be attached to one user, `AuthorId` should be added, and I suggest we only store `postId` to find likes and comments. `LikeId` and `CommentID` is not necessary in this table